subsys/modem/modem_cmux.c

```diff
@@ -435,6 +446,40 @@ static void modem_cmux_on_control_frame_uih(struct modem_cmux *cmux)
 	}
 }
 
+static void modem_cmux_conect_response_transmit(struct modem_cmux *cmux)
```

Comment:
Should be `_connect_`

Reply:
I will be fix this after my bicycle ride. Thanks